paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -76,7 +77,10 @@ class PaparazziPlugin : Plugin<Project> {
         project.tasks.named("merge${variantSlug}Assets") as TaskProvider<MergeSourceSetFolders>
       val mergeAssetsOutputDir = mergeAssetsProvider.flatMap { it.outputDir }
       val reportOutputDir = project.layout.buildDirectory.dir("reports/paparazzi")
-      val snapshotOutputDir = project.layout.projectDirectory.dir("src/test/snapshots")
+
+      val snapshotOutputDir =
+          project.extensions.extraProperties.properties["paparazzi_snapshot_dir"] as? File
```

Comment:
It should not be an arbitrary value in extraProperties. We should create a paparazzi plugin extension where this value is explicitly specified.

Reply:
That's a better idea - done now 💯 just need to fix up the unit tests now I think